### PR TITLE
chore: add simple smoke-tests

### PR DESCRIPTION
**Pull request by qgerome**

### CODE REVIEW

File: smoke-tests/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+cp .env.dist .env
+```
+
+3. Run the tests with a visual interface:
+```bash
+npm run test:ui # With a visual interface
+npm run test # Without a visual interface
+```
+
+## How to write smoke tests
```
Comment (toch) [inline smoke-tests/README.md:32]: FYI, that's something I haven't tried.

File: smoke-tests/.env.dist [hunk 2]
```diff
@@ -0,0 +1,3 @@
+OH_BASE_URL=
```
Comment (toch) [inline smoke-tests/.env.dist:1]: the acronym `OH` might not be obvious for a newcomer. I'd go for the full name.

File: smoke-tests/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,15 @@
+FROM mcr.microsoft.com/playwright:v1.45.0-jammy
+
+
+# Set the working directory
+WORKDIR /usr/src/app
+
+# Copy your project files into the Docker container
+COPY . .
```
Comment (toch) [inline smoke-tests/Dockerfile:8]: you might copy some temporary directories such as `node_modules`. I would copy the minimum you need to prepare the deps. So I think `package.json` should be enough, isn't it?

File: smoke-tests/README.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+## How to run smoke tests with docker
+
+1. Build the docker image:
+```bash
+docker build -t smoke-tests .
+```
+
+2. Run the docker container:
+```bash
+docker run  -it -v "$(pwd):/usr/src/app" smoke-tests
```
[the remark this answers is not in the document]
Comment (toch) [reply smoke-tests/README.md:12]: also I'd put all of that in a script. I can do that in a subsequent PR if you'd like

File: smoke-tests/playwright.config.js [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+    //   name: 'Microsoft Edge',
+    //   use: { ...devices['Desktop Edge'], channel: 'msedge' },
+    // },
+    // {
+    //   name: 'Google Chrome',
+    //   use: { ...devices['Desktop Chrome'], channel: 'chrome' },
+    // },
+  ],
+
+  /* Run your local dev server before starting the tests */
```
Comment (toch) [inline smoke-tests/playwright.config.js:78]: to remove ?

File: smoke-tests/tests/simple.spec.js [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+});
+
+test("view workspace's files", async ({ page }) => {
+  await page.getByRole("navigation").getByRole("link").nth(1).click(); // Files link
+  await expect(page.getByRole('columnheader', { name: 'Size' }).first()).toBeVisible();  
+  await expect(page.getByRole('columnheader', { name: 'Name' }).first()).toBeVisible();  
+  
+});
+
+test("view workspace's jupyterlab environment", async ({ page }) => {
```
Comment (toch) [inline smoke-tests/tests/simple.spec.js:39]: it catches an issue I have. my local hosting setup doesn't spawn jupyterhub. So the smoke test detects what it should ;).

**Review by toch — COMMENTED** [review]
Very useful, thank you.

A few comments, the important ones are:

* the copy of host files in the docker image that would restrict to the minimum
* the command to run the test with a container that does not work as is due to network consideration

File: smoke-tests/README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (qgerome) [reply smoke-tests/README.md:12]: I've updated the command. I wasn't so sure about the script as I did not want to hide that from the user but be my guest to write a script :)

File: smoke-tests/README.md (hunk 1 above)
Comment (qgerome) [reply smoke-tests/README.md:32]: Yes, it was more documented in case you wanted to add quickly other tests.

**Pull request merged**